afval.map

```diff
@@ -3108,4 +3108,845 @@ MAP
     END
   END
 
+#-----------------------------------------------------------------------------
+
+  LAYER
+    NAME                    "pand_loopafstand"
+    GROUP                   "afvalcontainers"
+    INCLUDE                 "connection_dataservices.inc"
+    DATA                    "geometrie FROM ( 
+                            SELECT bol.id
+                            , bol.geometrie
+                            , lac.loopafstand_categorie_omschrijving
+                            FROM public.huishoudelijkafval_bag_object_loopafstand bol
+                            INNER JOIN public.huishoudelijkafval_loopafstand_categorie lac
+                            ON bol.loopafstand_categorie_id = lac.id
+                            WHERE 1=1 
+                            AND bol.fractie_omschrijving = 'Rest'
+                            ) rest USING srid=28992 USING UNIQUE id"
+    TYPE                    POLYGON
+    MINSCALEDENOM           10
+    MAXSCALEDENOM           400000
+
+    PROJECTION
+      "init=epsg:28992"
+    END
+
+    METADATA
+      "wfs_title"           "pand loopafstand tot dichtsbijzijnde container in meters"
+      "wfs_srs"             "EPSG:28992"
+      "wfs_adstract"        "pand loopafstand tot dichtsbijzijnde container in meters"
+      "wfs_enable_request"  "*"
+      "gml_featureid"       "ID"
+      "gml_include_items"   "all"
+      "gml_types"           "auto"
+      "wms_title"           "Rest container"
+      "wms_enable_request"  "*"
+      #"wms_extent"          "4.58565 52.03560  5.31360 52.48769"
```

Comment:
In ander RD mapfiles staat bij wms_extent : "wms_extent"          "100000 450000 150000 500000"

Reply:
Check!